src/org/omegat/core/team2/RemoteRepositoryProvider.java

```diff
@@ -438,6 +460,40 @@ public void copyFromRepoToProject() throws IOException {
                 copyFile(from, to, null);
             }
         }
+      //
+      // This function is only used to copy remote omegat.project to local omegat.project.NEW
+      //
+        public void copyFromRepoToProjectTempNew() throws IOException {
+            if (!matches()) {
```

Comment:
Same here for my previous comment that we should avoid copy-and-modify.
We can consolidate a logic into the function.

```java
        public void copyFromRepoToProject() throws IOException {
            copyFromRepoToProject("");
        }

        public void copyFromRepoToProject(final String target) throws IOException {
```

```java
            File to = new File(projectRoot, withoutLeadingSlash(repoMapping.getLocal() + target));
```